limitations.md

```diff
@@ -0,0 +1,13 @@
+# Limitations in iceoryx
+- limited Windows 10 support [\#33](https://github.com/eclipse-iceoryx/iceoryx/issues/33)
+- no support for 32-bit platforms, use at your own risk
+- Duplicated service description entries within gateway config toml file should not result in duplication of service entries [\#574](https://github.com/eclipse-iceoryx/iceoryx/issues/574)
+- Possible double free of memory chunks on 32-bit systems [\#562](https://github.com/eclipse-iceoryx/iceoryx/issues/562)
+- Fix resource leak caused by move ctor/assignment [\#542](https://github.com/eclipse-iceoryx/iceoryx/issues/542)
```

Comment:
If this list is auto-generated, I suggest introducing a convention to have the name of bug issues describe the bug/limitation rather than a task to be done like this one. 
Otherwise, this list reads a bit awkwardly, like with this entry.